core/tee/tee_svc_cryp.c

```diff
@@ -427,17 +480,33 @@ static const struct tee_cryp_obj_type_props tee_cryp_obj_props[] = {
 		sizeof(struct rsa_keypair),
 		tee_cryp_obj_rsa_keypair_attrs),
 
-	PROP(TEE_TYPE_DSA_PUBLIC_KEY, 64, 512, 1024,
+	PROP(TEE_TYPE_DSA_PUBLIC_KEY, 64, 512, 3072,
 		sizeof(struct dsa_public_key),
 		tee_cryp_obj_dsa_pub_key_attrs),
 
-	PROP(TEE_TYPE_DSA_KEYPAIR, 64, 512, 1024,
+	PROP(TEE_TYPE_DSA_KEYPAIR, 64, 512, 3072,
 		sizeof(struct dsa_keypair),
 		tee_cryp_obj_dsa_keypair_attrs),
 
 	PROP(TEE_TYPE_DH_KEYPAIR, 1, 256, 2048,
 		sizeof(struct dh_keypair),
 		tee_cryp_obj_dh_keypair_attrs),
+
+	PROP(TEE_TYPE_ECDSA_PUBLIC_KEY, 1, 192, 521,
```

Comment:
Agree but no way to currently define multiple transient object properties because of lack algorithm informations (i.e use of TEE_AllocateTransientObject w/ TEE_TYPE_DSA_PUBLIC_KEY  it is not possible if it is for ALG_DSA_SHA1 or ALG_DSA_SHA224 or ALG_DSA_SHA256). The object type and the maximum key size must be specified so that all the container resources can be pre-allocated.